src/main/scala/mesosphere/marathon/core/task/update/impl/steps/TaskStatusEmitterPublishStepImpl.scala

```diff
@@ -12,13 +14,22 @@ import scala.concurrent.Future
 /**
   * Forward the update to the taskStatusEmitter.
   */
+// TODO(PODS): is the emitter stuff still in use?
```

Comment:
I'm hoping we can remove rxScala and use a lot more of akka streaming. I think a bunch of stuff we're currently either using even stream for or actors could be replaced with streaming.